docs/cosa-significa-essere-un-dev/README.md

```diff
@@ -0,0 +1,97 @@
+
+## L'empatia: la capacità di impersonare il prossimo
+
+Forse ti starai chiedendo cosa c'entri una qualità come l'empatia in un percorso di apprendimento e crescita professionale in un settore come quello della programmazione.
+
+Se ad un primo impatto potrebbe sembrare un dubbio legittimo, la verità è che l'empatia in questo caso gioca un ruolo fondamentale.
+
+Durante il tuo percorso, presto o tardi ti capiterà di imbatterti in un problema a cui non saprai far fronte; a quel punto, molto probabilmente il tuo primo impulso sarà quello di cercare in rete una soluzione e, pensa un po', la troverai!
+
+Penserai "_Che fortuna che ho avuto!_", ed è qui che sbaglieresti...**sbaglieresti di grosso**.
+
+Quella soluzione, che molto probabilmente ti aiuterà a risolvere il tuo problema, è lì per un motivo ben preciso: qualcuno, prima di te, ha avuto il tuo stesso problema; al contrario di te però, quel qualcuno non è stato "_così fortunato_" da trovare una soluzione online, ma è stato abbastanza empatico da mettersi nei panni di chi, come te, quel problema prima o poi lo avrebbe avuto; una volta trovata la soluzione quindi, si è adoperato per metterla a disposizione di tutti, **gratuitamente** e in maniera del tutto spontanea.
+
+Questo, caro lettore, è uno dei principi fondamentali su cui verte questo libro: **la condivisione libera del sapere (in questo caso collettivo) con chiunque ne abbia bisogno**.
+
+Ed è proprio per questo che ti parlo di empatia: durante il tuo percorso, che sia grazie a questo libro o ad altro, apprenderai un'enorme quantità di informazioni; mentre lo farai, rifletti sul perché ti è possibile farlo, e domandati se le nozioni da te apprese potranno in futuro essere utili a chi muoverà i primi passi in questo settore, proprio come stai facendo tu ora grazie al manoscritto che stai leggendo.
+
+Forte di questa consapevolezza, prendi coraggio e sii empatico: pubblica, ovunque tu voglia e in qualsiasi formato tu preferisca, ciò che hai imparato: per te sarà un notevole esercizio di ripasso e pratica sul campo; chiunque fruirà dei tuoi contenuti te ne sarà grato e sarà propenso a diventare un membro della tua community, ma sopra ogni cosa sarà ispirato a condividere a sua volta le nozioni che imparerà lungo il suo percorso...proprio come hai fatto tu prima di lui.
+
+Ciò che vogliamo trasmetterti è la propensione ad adottare uno spirito di condivisione: oltre ad essere il cuore pulsante del movimento Open Source, e quindi del libro che stai leggendo, è anche un principio nobile e al contempo stesso utile; qualora non ne comprenderai subito il senso ed il potenziale, o magari ti sembrerà una perdita di tempo, tieni a mente quanto segue: arriverà il giorno in cui ringrazierai chi, prima di te, ha seguito il nostro consiglio.
+
+Fidati di noi: fidati di chi, prima di te, è stato aiutato e che oggi è qui a condividere il suo sapere con te.
```

Comment:
Questi ultimi 4 paragrafi, presi singolarmente e anche nel loro insieme, mi piacciono molto, ma non vorrei risultassero troppo ripetitivi. È più una domanda che pongo a chi leggerà queste parole e questa review: Si sta dando enfasi al concetto in maniera corretta o "un pochino troppo"? Ripeto, non è la scelta delle parole, che approvo al 100%; ma alla ripetizione del concetto.

Reply:
Rileggendolo credo sia sufficiente rimuovere il "fidati di noi.... con te" per non esasperare il concetto.